resources/views/auth/login.blade.php

```diff
@@ -34,15 +38,26 @@
 
             <!-- Remember Me -->
             <div class="block mt-4">
-                <label for="remember_me" class="inline-flex items-center">
+                <label for="remember_me" class="inline-flex items-center pb-2">
                     <input id="remember_me" type="checkbox" class="rounded border-gray-300 text-indigo-600 shadow-sm focus:border-indigo-300 focus:ring focus:ring-indigo-200 focus:ring-opacity-50" name="remember">
-                    <span class="ml-2 text-sm text-gray-600">{{ __('Remember me') }}</span>
+                    <span class="ml-2 text-sm text-gray-600 ">{{ __('Remember me') }}</span>
                 </label>
             </div>
                 <x-button class="ml-3">
                     {{ __('Log in') }}
                 </x-button>
             </div>
+            <div class="w-120 rounded-lg sm:max-w-md mt-6 px-8 py-4 bg-white shadow-md overflow-hidden sm:rounded-lg">
+        <!-- Session Status -->
+       <div class = "text-s font-normal text-center pb-2">Your Login instructions as follows</div>
+        <div class="text-xs"><ul class="list-disc justify-content ">
+            <li >For the username please use your account name of your NSBM student email address.</li>
+            <li >Account name is the first portion of your NSBM student email address before the “@” symbol.</li>
+
+<li >For further queries, please drop us an email to NSBM IT Support via <a>its@nsbm.ac.lk.<a/></li></ul></div>
```

Comment:
Remove extra space and use correct indentation

Reply:
ok